dlgr/griduniverse/game_config.yml

```diff
@@ -9,38 +21,148 @@ player_config:
     PURPLE: [0.85, 0.60, 0.85]
     TEAL: [0.77, 0.96, 0.90]
 
+
 item_defaults:
+  # Each item definition must include a unique item_id. This actual value doesn't matter,
+  # though an easily identifiable label can be helpful in developing and
+  # debugging your experiment.
   item_id: default
+
+  # How many instances of this item should the world initially include?
+  # If "respawn" (see below) is true, consumed items will be replaced to maintain
+  # this number of instances. Note that this value may increase over the course
+  # of a game depending on the values of seasonal_growth_rate and spawn_rate (see below).
   item_count: 8
+
+  # How many calories does a single instance provide when a player consumes it?
+  # In the simple case, the consuming player will get all the caloric benefit from
+  # items they consume, but there are also options for dividing this benefit among
+  # other players (see public_good_multiplier, below).
   calories: 0
+
+  # Can a player co-occupy the grid block this item is sitting on?
   crossable: true
+
+  # Does a player need to explicitly interact with this item via the action button
+  # when they co-occupy its grid block, or do the immediately consume the item
+  # without needing to take any explicit action?
   interactive: false
```

Comment:
Maybe this should default to `true` since the `food` special case in the only place we're likely to want it to be `false`.